src/store/rewards/saga.ts

```diff
@@ -64,7 +64,7 @@ export function* syncZEROPrice() {
 }
 
 export function* syncRewardsAndTokenPrice() {
-  if (!featureFlags.enableRewards || featureFlags.enableMatrix) {
+  if (!featureFlags.enableRewards) {
```

Comment:
not sure if fetching the token price at regular intervals does anything for us right now, in matrix. since we haven't implemented rewards yet. but your call :)